### PR TITLE
feature: relates to #121; ensure operator-builder binary builds

### PR DESCRIPTION
**Pull request by scottd018**
Signed-off-by: Dustin Scott <sdustin@vmware.com>

### CODE REVIEW

**Comment by lander2k2** [comment]
LGTM.  Only comment is commit message should be `ci: ...` rather than `feature: ...`
I can change that when I squash and merge.

**Review by lander2k2 — APPROVED** [review]

**Comment by scottd018** [comment]
> LGTM. Only comment is commit message should be `ci: ...` rather than `feature: ...`
> I can change that when I squash and merge.

Gotcha, makes sense.  Once we get our developer guide we probably should include what we feel are the recommended categories probably via https://github.com/vmware-tanzu-labs/operator-builder/issues/64 .

**Comment by JefeDavis** [comment]
@lander2k2 @scottd018 `ci` is a normal category according to conventions, it describes changes to a CI pipeline that do not add or fix components of the software itself

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+name: Lint
```
Comment (JefeDavis) [inline .github/workflows/test.yaml:2]: should be changed to Test

Comment (scottd018) [reply .github/workflows/test.yaml:2]: Good catch @JefeDavis . I’m away from my computer for a few days. Any way I can convince you to push the change up so we can get it merged?

**Comment by scottd018** [comment]
> @lander2k2 @scottd018 `ci` is a normal category according to conventions, it describes changes to a CI pipeline that do not add or fix components of the software itself

Yep, that’s good info. I just ask that we link what is considered common convention in the extended contributing guide. Not everyone has the conventions memorized so a link will be handy for those that don’t know them well or are learning to conform to them (like myself…ha)

**Pull request merged**